tempus_fugit_minecraft/model.py

```diff
@@ -487,18 +489,41 @@ def handle_speed_change(self, increase: bool) -> None:
         """!
         @brief Handles the speed change event
         @param increase A boolean indicator of whether we increase or decrease the speed
+<<<<<<< HEAD
         @see [Issue#68](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/68)
+=======
+        @see issue 68
+>>>>>>> 0ad6578 (Doxygen comment+ jump_speed upgrade)
         """
         if increase:
-            self.player.speed_up()
+            self.player.increase_speed()
         else:
-            self.player.speed_down()
+            self.player.decrease_speed()
 
     def handle_jump(self) -> None:
         """!
         @brief Handles the jump event
+<<<<<<< HEAD
         @see [Issue#68](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/68)
+=======
+        @param None
+        @see issue 68
+>>>>>>> 0ad6578 (Doxygen comment+ jump_speed upgrade)
```

Comment:
Like above, This PR is blocked until this merge conflict is resolved. We need to accept the current change for this diff.